final_application/fill_handwriting_with_background.py

```diff
@@ -0,0 +1,90 @@
+import torch
+import yaml
+from pathlib import Path
+from PIL import Image
+import argparse
+from analysis_segmenter import AnalysisSegmenter
+from typing import Union
+from tqdm import tqdm
+from torchvision import transforms
+import torch.nn as nn
+import copy
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--input-dir', type=Path, help='the directory of the images you want to remove the handwriting from')
+parser.add_argument('--output-dir', type=Path, help='the directory of the images without handwriting')
+args = parser.parse_args()
+
+
+def is_image(file_name: Union[str, Path]) -> bool:
+    if not isinstance(file_name, Path):
+        file_name = Path(file_name)
+    return file_name.suffix.lower() in ['.png', '.jpeg', '.jpg', '.svg']
+
+
+def inpaint_patch(segmentation: Image, ground_truth: torch.Tensor, ) -> torch.Tensor:
+    max_pooling = nn.MaxPool2d(7, 1, 3)
+    to_tensor = transforms.ToTensor()

```

Comment:
Consider using `torch.transforms.functional`. (When used I often see it imported like this ` import torch.transforms.functional as F`